src/main/java/uk/nhs/prm/repo/ehrtransferservice/exceptions/TransferUnableToUpdateException.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.prm.repo.ehrtransferservice.exceptions;
+
+import uk.nhs.prm.repo.ehrtransferservice.exceptions.base.DatabaseException;
+
+import java.util.UUID;
+
+public class TransferUnableToUpdateException extends DatabaseException {
```

Comment:
```suggestion
public class UnableToUpdateConversationException extends DatabaseException {
```

Maybe be more explicit here.